### PR TITLE
Add unit tests for validators v4

### PR DESCRIPTION
**Pull request by pyasi**
### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Added tests for the ValidateAPIKey class. A previously untested piece of functionality
- Made small changes in the ValidateAPIKey class to conform to project standards

### CODE REVIEW

**Comment by pyasi** [comment]
@misterdorm I just made a new PR against v4. Would you mind reapplying the labels and accepting? Thanks for the advice!

**Comment by misterdorm** [comment]
Replaces #705

File: test/test_validators.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def test_validate_message_dict_text_edge_case_passes(self):
+        validator = ValidateAPIKey(regex_strings=['test_one'])
+        test_message = "test_two"
+        validator.validate_message_dict(test_message)
+
+    def test_validate_message_dict_text_edge_case_raises_exception(self):
+        validator = ValidateAPIKey(regex_strings=['test_one'])
+        test_message = "test_one"
+        with self.assertRaises(APIKeyIncludedException):
+            validator.validate_message_dict(test_message)
```
Comment (42B) [inline test/test_validators.py]: Maybe add a newline at the end of the file here?

**Comment by thinkingserious** [comment]
Hello @pyasi,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request closed without merging**